packages/frama-c/frama-c.32.0~beta/opam

```
@@ -0,0 +1,203 @@
+  "odoc" { with-doc }
+  "unionFind" { >= "20220109" }
+  "why3" { >= "1.8.2" & < "1.9.0" }
+  "yaml" { >= "3.0.0" }
+  "yojson" {>= "1.6.0" & (>= "2.0.1" | !with-test)}
+  "zarith" { >= "1.13" }
+
+  # PPXs
+  "ppxlib" { >= "0.33.0" }
+  "ppx_deriving"
+  "ppx_deriving_yojson"
+  "ppx_deriving_yaml" { >= "0.2.0" }
+  "ppx_inline_test"
+
+  # GTK3 disabled on macOS (segfaults), and made optional on Windows
+  # (due to complex situation with Cygwin + MinGW).
+  "lablgtk3" { >= "3.1.0" & os!="macos" & os-family!="windows" }
+  "lablgtk3-sourceview3" { os!="macos" & os-family!="windows" }
+  "conf-gtksourceview3" { os!="macos" & os-family!="windows" }
+]
+
+# Note: do not put particular versions here, if some version is *incompatible*,
+# use the field 'conflicts'.
+depopts: [
+  "apron"
+  "zmq"
+  "lablgtk3" { os-family="windows" }
+  "lablgtk3-sourceview3" { os-family="windows" }
+  "conf-gtksourceview3" { os-family="windows" }
+]
+
+conflicts: [
+  "cairo2" { < "0.6.2" }
+  "mlmpfr" { < "4.1.0-bugfix2" }
+  "pilat"  { <= "1.6" }
+  "result" { < "1.5" }
+]
+
+post-messages: [
+"The Frama-C/WP plug-in requires one or more external prover(s).
+Recommended provers are:
+- Alt-Ergo (https://alt-ergo.ocamlpro.com)
+- CVC4     (https://cvc4.github.io)
+- CVC5     (https://cvc5.github.io)
+- Z3       (https://github.com/Z3Prover/z3)
+Use 'why3 config detect' to configure new provers.
+ " { success }
+"Ivette is a new GUI for Frama-C, currently in development.
+Run 'ivette' once to finalize installation (requires an internet connection).
+Once finalized, 'ivette' will work offline.
+Finalization also requires Node v20 or v22 and Yarn:
+- install NVM (https://github.com/nvm-sh/nvm)
+- run 'nvm install 22'
+- run 'nvm use 22'
+- run 'npm install --global yarn'" { success }
+]
+
+url {
+  src: "https://www.frama-c.com/download/frama-c-32.0-beta-Germanium.tar.gz"
+  checksum: "sha256=868d57ef8007fe6c0836cd151d8c294003af34aa678285eff9547662cad36aa3"
+}
```

```suggestion
}
x-ci-accept-failures: [  # the following either uses musl or has an incompatible gcc version
  "alpine-3.22"
  "centos-9"
  "debian-11"
  "debian-12"
  "opensuse-15.6"
  "ubuntu-22.04"
]
```

Thanks for investigating these failures! I've had a brief skim of the logs and tend to agree.
As a compromise, I suggest silencing the errors with the above to reduce noise from the CI.

The `Failed to get sources of alt-ergo.1.01: curl failed` persists, despite me restarting the failing CI jobs a couple times yesterday (also to eliminate noise). Anyway, that is an `alt-ergo` issue :shrug: